af/fastpath/fastpath/s3feeder.py

```diff
@@ -154,11 +159,15 @@ def _cb(bytes_count):
         # TODO: handle missing file
         log.info("Downloading can %s size %d MB" % (fn, size / 1024 / 1024))
         diskf.parent.mkdir(parents=True, exist_ok=True)
+        tmpf = diskf.with_suffix(".s3tmp")
```

Comment:
Do you think it makes sense to add something to clean these files up in the event of the process being restarted or killed?

Reply:
This is meant for manual runs and there is no automated cleanup of the cached files (given that the pipeline does not know if we want to use the cached files only once). At the moment I'm deleting the whole cache manually.